public/app/features/admin/UserAdminPage.tsx

```diff
@@ -103,13 +101,16 @@ export class UserAdminPage extends PureComponent<Props> {
   };
 
   render() {
-    const { navModel, user, orgs, sessions, ldapSyncInfo, isLoading } = this.props;
+    const { user, orgs, sessions, ldapSyncInfo, isLoading } = this.props;
     const isLDAPUser = user && user.isExternal && user.authLabels && user.authLabels.includes('LDAP');
     const canReadSessions = contextSrv.hasPermission(AccessControlAction.UsersAuthTokenList);
     const canReadLDAPStatus = contextSrv.hasPermission(AccessControlAction.LDAPStatusRead);
+    const pageNav: NavModelItem = {
```

Comment:
not sure how we feel about creating this manually instead of adding it to the redux store? we have a mix of both patterns at the moment 😕